tasks/polarisTask/util/PolarisPlatformSupport.ts

```diff
@@ -6,7 +6,22 @@ export default class PolarisPlatformSupport {
         if (platform == "win32") {
             return "/api/tools/" + client + "_cli-win64.zip";
         } else if (platform == "darwin") {
-            return "/api/tools/" + client + "_cli-macosx.zip";
+            // The mac runner architecture in the ADO returns same for both arm and non arm machines.
+            // So, we fetch the CPU information and try to detect if it is a non-arm runner
+            const cpuInfo = os.cpus();
+            if ((cpuInfo.length) > 0) {
```

Comment:
minor: `if (cpuInfo.length > 0)` would still be the same I think(no need of extra brackets may be)

Reply:
This language requires the outer braces, otherwise it shows error.

Reply:
sorry I meant the inner braces that enclose `cpuInfo.length`

Reply:
Oh okay! Will keep in mind to try that out if I make any further changes to this repo. Thanks!